include/clang/AST/Type.h

```diff
@@ -3689,6 +3693,35 @@ class UnresolvedUsingType : public Type {
 };
 
 
+class TypeVariableType : public Type, public llvm::FoldingSetNode {
```

Comment:
Sorry I missed that.  Could you move it to be after the Profile method, so that it matches the order in most of the other definitions of classes for types?